sasl/plain/src/main/java/org/wildfly/security/sasl/plain/PlainSaslClient.java

```diff
@@ -44,11 +44,13 @@
 
     private final String authorizationId;
     private final CallbackHandler cbh;
+    private final long passwordProfile;
     private boolean complete = false;
 
-    PlainSaslClient(final String authorizationId, final CallbackHandler cbh) {
+    PlainSaslClient(final String authorizationId, final CallbackHandler cbh, boolean skipNormalization) {
```

Comment:
Just minor, this new parameter can be made final, i.e.,

```final boolean skipNormalization```

Reply:
done